### DISCUSSION TITLE
The QuerySet value for an exact lookup must be limited to one result using slicing

### DISCUSSION THREAD

**Discussion by LRafael21** [netbox-community/netbox#9555] [Q&A]
<class 'ValueError'>  The QuerySet value for an exact lookup must be limited to one result using slicing.

i got this  Server Error since the version 2.7.12, when i want to access to my regions, can somebody help me? thank you

<img width="504" alt="server error" src="https://user-images.githubusercontent.com/97407550/174313739-59550a1b-32d4-438c-a806-e9f62686e9cc.PNG">

**Comment by candlerb** [comment]
Please can you show which actual URL you were accessing (as seen in the browser URL bar), and also obtain the full backtrace (set EMAIL and ADMINS in configuration.py to get this emailed to you)

However, if this is a bug in Netbox, it's unlikely that it will be fixed in such an old version.  If you can't upgrade, you may need to find a local Python programmer who's happy to do it for you.
